hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DatanodeInfo.java

```diff
@@ -49,7 +51,7 @@ public class DatanodeInfo extends DatanodeDetails {
   private volatile long lastHeartbeatTime;
   private long lastStatsUpdatedTime;
   private int failedVolumeCount;
-
+  private List<VolumeInfoProto> volumeInfos;
```

Comment:
`volumeInfos` is never assigned.  I think the intention was to update this variable in `updateStorageReports` in the block holding `writeLock`.

Reply:
Thank you for pointing out this issue! I hadn't noticed these details before.